R/BFF_Demux.R

```diff
@@ -389,203 +276,18 @@ GenerateCellHashCallsBFF <- function(barcodeMatrix, assay = "HTO", min_average_r
 }
 
 
-BFFDemux <- function(seuratObj, assay, calling_paramset) {
+BFFDemux <- function(seuratObj, assay, simple_threshold, doublet_thresh, neg_thresh, pos_dist, neg_dist) {
```

Comment:
my R is not necessarily good enough to know the difference, but I tend to like named args in this scenario. it's more verbose, but "simple_threshold = simple_threshold, doublet_thresh = doublet_thresh", etc. is probably less likely to break in a hard to detect manner if you ever change the downstream code.